omnibus/files/private-chef-cookbooks/private-chef/spec/libraries/helper_spec.rb

```diff
@@ -114,6 +114,64 @@
     end
 
     context 'when elastic search is unavailable' do
+      it 'should return 2 but the request fails first time only' do
```

Comment:
The tests look good, but I'm thinking they could be a shared example that exists locally this context, and uses `let` or similar  to set number of failures?